### PR TITLE
Update more Docs

### LINKED ISSUE #633: Some GATK < 4 processes are still present in documentation and config files.
**Describe the bug**
These processes are still present in documentation and configuration:

- [x] IndelRealigner
- [x] RealignerTargetCreator

There might be other removed processes that are still left in the documentation/configuration. I will have a look at this when I get time.

### PR DESCRIPTION
**Pull request by MaxUlysse**
- Closes #633 
- Clean comments
- Clean docs
- rework the version gathering system

## PR checklist
 - [X] PR is made against `dev` branch
 - [X] This comment contains a description of changes (with reason)
 - [X] Ensure the test suite passes (`./scripts/test.sh -p docker -t ALL`).
 - [x] `CHANGELOG.md` is updated
 - [X] `README.md` is updated

### CODE REVIEW

File: docs/USAGE.md [hunk 1]
```diff
@@ -93,28 +93,33 @@ Choose an output directory
 
 ### --project `ProjectID`
 
-Specify a project number ID on a UPPMAX cluster. (optionnal if not on such a cluster)
+Specify a project number ID on a UPPMAX cluster.
+(optionnal if not on such a cluster)
```
Comment (alneberg) [inline docs/USAGE.md]: *optional

File: docs/USE_CASES.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # Use cases
 
-The workflow has three pre-processing options: `mapping`, `realign` and `recalibrate`. Using the `mapping` directive one will have a pair of mapped, deduplicated and recalibrated BAM files in the `Preprocessing/Recalibrated/` directory. Furthermore, during this process a deduplicated BAM file is created in the `Preprocessing/NonRealigned/` directory. This is the usual option you have to give when you are starting from raw FASTQ data:
+The workflow has three pre-processing options: `mapping`, `realign` and `recalibrate`.
```
Comment (alneberg) [inline docs/USE_CASES.md]: Realign should be removed here right?

File: docs/USE_CASES.md [hunk 3]
```diff
@@ -93,7 +112,8 @@ SUBJECT_ID  XX    1    SAMPLEIDR    9    /samples/relapse9_1.fastq.gz    /sample
 
 ## Starting from realignement
```
Comment (alneberg) [inline docs/USE_CASES.md]: See if this paragraph and the one below should be removed?

File: docs/USAGE.md (hunk 1 above)
Comment (MaxUlysse) [reply docs/USAGE.md]: :man_facepalming:

File: docs/USE_CASES.md (hunk 2 above)
Comment (MaxUlysse) [reply docs/USE_CASES.md]: Good point, I'll update that as well

File: docs/USE_CASES.md (hunk 3 above)
Comment (MaxUlysse) [reply docs/USE_CASES.md]: Same

**Comment by alneberg** [comment]
👍 👏

**Comment by MaxUlysse** [comment]
So now, where can we find some beginners to help us with this USAGE.md docs?

**Comment by alneberg** [comment]
I can give it a go. But it would need some careful proofreading for factual errors.

**Pull request merged**